src/matchers/utils.tsx

```diff
@@ -121,3 +121,69 @@ export function formatMessage(
 function formatValue(value: unknown) {
   return typeof value === 'string' ? value : stringify(value);
 }
+
+function display(value: unknown) {
```

Comment:
I think all of the code added here can be removed if you use checkHostElement instead